ui/app/models/task-state.js

```diff
@@ -9,13 +9,18 @@ export default Fragment.extend({
 
   name: attr('string'),
   state: attr('string'),
+  kind: attr('string'),
   startedAt: attr('date'),
   finishedAt: attr('date'),
   failed: attr('boolean'),
 
   isActive: none('finishedAt'),
   isRunning: and('isActive', 'allocation.isRunning'),
 
+  isConnectProxy: computed('kind', function() {
+    return (this.kind || '').startsWith('connect-proxy:');
```

Comment:
Do you need a `get` here? Not sure if you've migrated to get-less ember yet?

Reply:
I found [this commit](https://github.com/hashicorp/nomad/commit/f6232f181729915a08b8e57325080792462dc266) that uses getters, though not exhaustively, so it seems to be at least present in the codebase.